modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala

```diff
@@ -129,45 +129,56 @@ class ObserveCommandRoutes[F[_]: Async: Compression](
     //   se.setObserver(inputQueue, obsId, user, obs) *>
     //     Ok(s"Set observer name to '${obs.value}' for sequence $obsId")
     //
-    // case POST -> Root / ObsId(obsId) / "tcsEnabled" / BooleanVar(tcsEnabled) as user =>
-    //   se.setTcsEnabled(inputQueue, obsId, user, tcsEnabled) *>
-    //     Ok(s"Set TCS enable flag to '$tcsEnabled' for sequence $obsId")
-    //
-    // case POST -> Root / ObsId(obsId) / "gcalEnabled" / BooleanVar(gcalEnabled) as user =>
-    //   se.setGcalEnabled(inputQueue, obsId, user, gcalEnabled) *>
-    //     Ok(s"Set GCAL enable flag to '$gcalEnabled' for sequence $obsId")
-    //
-    // case POST -> Root / ObsId(obsId) / "instEnabled" / BooleanVar(instEnabled) as user =>
-    //   se.setInstrumentEnabled(inputQueue, obsId, user, instEnabled) *>
-    //     Ok(s"Set instrument enable flag to '$instEnabled' for sequence $obsId")
-    //
-    // case POST -> Root / ObsId(obsId) / "dhsEnabled" / BooleanVar(dhsEnabled) as user =>
-    //   se.setDhsEnabled(inputQueue, obsId, user, dhsEnabled) *>
-    //     Ok(s"Set DHS enable flag to '$dhsEnabled' for sequence $obsId")
+    case req @ POST -> Root / ObsIdVar(obsId) / ClientIDVar(clientId) / "tcsEnabled" /
```

Comment:
Should we consider passing the `clientId` in a header instead?

Reply:
I don't know any pros/cons?